apps/src/code-studio/initApp/project.js

```diff
@@ -280,6 +285,10 @@ var projects = module.exports = {
     return this.exceedsAbuseThreshold();
   },
 
+  hideBecauseSharingDisabled() {
+    return this.getSharingDisabled();
+  },
```

Comment:
why not just call `project.getSharingDisabled()` instead of introducing this method?